[2024-02-29 14:37:13,005] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_redshift.crear_tabla_redshift scheduled__2022-09-25T03:00:00+00:00 [queued]>
[2024-02-29 14:37:13,037] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_redshift.crear_tabla_redshift scheduled__2022-09-25T03:00:00+00:00 [queued]>
[2024-02-29 14:37:13,039] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-02-29 14:37:13,040] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2024-02-29 14:37:13,040] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-02-29 14:37:13,076] {taskinstance.py:1397} INFO - Executing <Task(RedshiftSQLOperator): crear_tabla_redshift> on 2022-09-25 03:00:00+00:00
[2024-02-29 14:37:13,080] {standard_task_runner.py:52} INFO - Started process 799 to run task
[2024-02-29 14:37:13,089] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_con_conexion_redshift', 'crear_tabla_redshift', 'scheduled__2022-09-25T03:00:00+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/dag_postgres_database.py', '--cfg-path', '/tmp/tmpwhbh0wgv', '--error-file', '/tmp/tmpgyto2rgo']
[2024-02-29 14:37:13,091] {standard_task_runner.py:80} INFO - Job 89: Subtask crear_tabla_redshift
[2024-02-29 14:37:13,172] {task_command.py:371} INFO - Running <TaskInstance: dag_con_conexion_redshift.crear_tabla_redshift scheduled__2022-09-25T03:00:00+00:00 [running]> on host 85ed7e7789a6
[2024-02-29 14:37:13,339] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Joaco
AIRFLOW_CTX_DAG_ID=dag_con_conexion_redshift
AIRFLOW_CTX_TASK_ID=crear_tabla_redshift
AIRFLOW_CTX_EXECUTION_DATE=2022-09-25T03:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-25T03:00:00+00:00
[2024-02-29 14:37:13,339] {redshift_sql.py:76} INFO - Executing statement: 
            create table if not exists fin_mundo(
                dt date,
                pais varchar(30)
            )
        
[2024-02-29 14:37:13,349] {base.py:68} INFO - Using connection ID 'redshift_keys' for task execution.
[2024-02-29 14:37:14,696] {dbapi.py:231} INFO - Running statement: 
            create table if not exists fin_mundo(
                dt date,
                pais varchar(30)
            )
        , parameters: None
[2024-02-29 14:37:15,329] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_con_conexion_redshift, task_id=crear_tabla_redshift, execution_date=20220925T030000, start_date=20240229T143713, end_date=20240229T143715
[2024-02-29 14:37:15,464] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-29 14:37:15,768] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
